i18n/locales/source/course_content/2020/express-2020.json

```diff
@@ -879,36 +712,36 @@
       "courseC_artist_loops5b_c": "Your program should look like this!\n\n[blockly block][0]"
     }
   },
-  "https://studio.code.org/s/express-2020/stage/7/puzzle/10": {
+  "https://studio.code.org/s/express-2020/stage/6/puzzle/10": {
```

Comment:
These changes make me nervous. Why did the URL change here?

Reply:
It looks like some levels were moved around in https://github.com/code-dot-org/code-dot-org/commit/bb3cfd2d09664ad5d589d4612a7598fa6c24d2de#diff-dcf7270bfad7b445c0600cb9fdbdfedb. Because only express and pre-express were affected (which are generally made up of levels from a-f), it doesn't seem like this had major side effects. I'm also not seeing major side effects in #35887

Reply:
We were probably saved from major side effects by these strings being duplicates; if they were not, we would have lost translations.

I'm mostly worried here because this is a sign that our system for confirming that a course is finalized before enabling it clearly isn't working, if major changes like this are coming in after we've frozen. @dmcavoy do you happen to have context for why this happened?

Reply:
I don't you should ask Mike and Aaron what might have changed. I was under the impression we were not making changes like this anymore